index.js

```diff
@@ -19,7 +19,8 @@ var materialPackages = [
   { name: '@material/menu', css: true, js: true },
   { name: '@material/toolbar', css: true, js: true },
   { name: '@material/tabs', css: true, js: true },
-  { name: '@material/ripple', css: true, js: true }
+  { name: '@material/ripple', css: true, js: true },
+  { name: '@material/linear-progress', css: true, js: true}
```

Comment:
Minor nitpick, but you're missing a space before the closing `}`.

We should install Prettier.js.